### ISSUE TITLE
[recclasses] Non-offline dependencies (Trac #1567)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1559]
Currently recclasses depends on portia (r142454) and ophelia(r142462).  This is breakage that needs to be fixed.  One of the main reasons for the creation of the project was to provide a project with minimal dependencies that people could include into any meta-project.  This project therefore can't have any dependencies outside of offline-software.

Can we remove those dependencies and get whatever fixes into portia and ophelia that are needed so that their tests pass on all platforms?

Migrated from https://code.icecube.wisc.edu/ticket/1567
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Currently recclasses depends on portia (r142454) and ophelia(r142462).  This is breakage that needs to be fixed.  One of the main reasons for the creation of the project was to provide a project with minimal dependencies that people could include into any meta-project.  This project therefore can't have any dependencies outside of offline-software.\n\nCan we remove those dependencies and get whatever fixes into portia and ophelia that are needed so that their tests pass on all platforms?\n\n", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[recclasses] Non-offline dependencies", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-02-25T16:34:10", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160226T20:05:16: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "new" to "closed"
* hdembinski commented:

> In changeset 142573:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142573"
> fixing circular dependency, closes #1559
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
